packages/eslint-config-godaddy-react/index.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     'react/jsx-uses-react': 1,
     'react/jsx-equals-spacing': 2,
     'react/prefer-es6-class': 2,
+    'react/prop-types': 0,
```

Comment:
> Could this be done as part of the next major version aka upgrade to eslint@v9?

It _could_ - though it's not a breaking change since it is removing a check, as opposed to adding a new one. What's your reasoning for waiting?

eslint@9 is probably going to be a big effort from what I've seen. Has anybody started into that work?